cmd/create/create.go

```diff
@@ -0,0 +1,77 @@
+package create
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+
+	"realcloud.tech/pligos/cmd/compile"
+	"realcloud.tech/pligos/pkg/helm"
+	"realcloud.tech/pligos/pkg/pligos"
+)
+
+type Config struct {
+	ConfigPath  string
+	ContextName string
+	ChartPath   string
+}
+
+func Create(config Config) error {
+	return createTree(config.ConfigPath, config.ContextName, config.ChartPath)
+}
+
+func NewHelm(configPath, contextName string) (*helm.Helm, error) {
```

Comment:
Seems that this method is in the wrong package too.
Besides, this method is the only one that calls helm.New().
Potentially, this should be in pligos/pkg/helm/helm.go